espnet2/enh/encoder/stft_encoder.py

```diff
@@ -65,6 +71,22 @@ def forward(self, input: torch.Tensor, ilens: torch.Tensor):
 
         return spectrum, flens
 
+    def reset_config(self):
+        """Reset the configuration of STFT window and hop lengths."""
+        self.reconfig_for_fs(self.default_fs)
+
+    def reconfig_for_fs(self, fs):
```

Comment:
How about making the `reconfig_for_fs` and `reset_config` functions private and call them in the `forward` function? I think that will be much clearer and safer. I feel changing the internal property of the `STFT` module somewhere else and forgetting to reset it may lead to hidden dangers. And just passing the `fs` to the `forward` function will be much simpler.

Reply:
Good idea!

Reply:
done